ocs_ci/ocs/must_gather/const_must_gather.py

```diff
@@ -739,7 +739,10 @@
     "ocs-storagecluster-rbdplugin-snapclass.yaml",
 ]
 
-if config.ENV_DATA["platform"].lower() == constants.OPENSHIFT_DEDICATED_PLATFORM:
+if (
```

Comment:
I think it would be better to use constants.MANAGED_SERVICE_PLATFORMS here and is similar situations below